sum.cpp

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+using namespace std;
+
+int sum(int , int );
+
+int main(){
+    int x, y;
+    cout << "inserisci il primo numero intero";
+    cin >> x;
+    cout << "inserisci il secondo numero intero";
+    cin >> y;
```

Comment:
non ti sembra ci sia un po' di ridondanza qui? potresti creare una funzione esterna per evitare di ripetere questa cosa due volte, un po' come ha fatto il tuo collega qui https://github.com/Helias/git-course-exercises/pull/12

Reply:
Ty so much for your advice! I will modify my code with something which work better than my last version and upload my minor improvements soon as possible. Have a nice day.

Reply:
I see you are a man of culture as well.